apps/src/blockly/addons/cdoCss.js

```diff
@@ -0,0 +1,23 @@
+export default function initializeCss(blocklyWrapper) {
+  blocklyWrapper.Css.register([
+    `.fieldGridDropDownContainer.blocklyMenu .blocklyMenuItem {
+      width: 32px;
+      height: 32px;
+      padding: 0px;
+      border: none;
+    }
+    .fieldGridDropDownContainer .blocklyMenuItem.blocklyMenuItemSelected {
+      background-color: white;
+    }
+    /* Change look of focus/highlighted cell */
+    .fieldGridDropDownContainer .blocklyMenuItem.blocklyMenuItemHighlight {
+      box-shadow: 0 0 0 4px hsla(0.57, 10%, 34%, .2);
+    }
+    .blocklyMenuItemContent > img {
+      width: 32px;
+      height: 32px;
+      object-fit: contain;
```

Comment:
Do we need this to work on IE11?  I don't think it's [supported](https://caniuse.com/?search=object-fit) there.

Reply:
Confirmed with @KylieModen that we're okay with styling things not looking right in IE11.